src/test/java/gov/healthit/chpl/aqa/stepDefinitions/ChplAPISteps.java

```diff
@@ -0,0 +1,24 @@
+package gov.healthit.chpl.aqa.stepDefinitions;
+import static org.junit.Assert.assertEquals;
```

Comment:
There should be some spaces between the package and the imports, and between the imports and the class definition. The imports should also be ordered. In Eclipse, `Ctrl-Shift-O` will make that happen automatically. Suggest doing so on all files.

Also, functions should have a space between them, generally, everywhere:
```
void doSomething() {
   ...
}
void doAnotherThing() {
   ...
}
```
should be
```
void doSomething() {
   ...
}

void doAnotherThing() { //see inserted space above?
   ...
}
```